huntsman/scheduler/observation.py

```diff
@@ -113,3 +113,73 @@ def __init__(self, position, pattern=dither.dice9, n_positions=9,
 
         # Specify directory root for file storage
         self._directory = os.path.join(self._image_dir, 'flats')
+
+
+class DarkObservation(Observation):
+
+    """ Collection of darks when weather is bad
+
+    Dark observations will consist of both multiple exposure, but no fields.
+    Also, the mount will be parked when used.
+
+    Note:
+        For now the new observation must be created like a normal `Observation`,
+        with one `exptime` and one `field`. Then use direct property assignment
+        for the list of `exptime` and `field`. New `field`/`exptime` combos can
+        more conveniently be set with `add_field`
+    """
+
+    def __init__(self, position, *args, **kwargs):
+        """
+        Args:
+            position (str): Center of field, can be anything accepted by
+                `~astropy.coordinates.SkyCoord`.
+        """
+        # Create the observation
+        dark_coords = position.to_string('hmsdms')
+        dark_field = Field('Dark Field', dark_coords)
```

Comment:
You should be able to just pass the `position` directly

```suggestion
        dark_field = Field('Dark Field', position)
```